job/client/src/main/java/alluxio/client/job/JobMasterClient.java

```diff
@@ -66,15 +66,15 @@ public static JobMasterClient create(JobMasterClientContext conf) {
   JobServiceSummary getJobServiceSummary() throws IOException;
 
   /**
-   * @return the list of ids of all jobs
+   * @return the list of all plan ids
```

Comment:
Is this wording because a Job can either be a task or plan and only top level plans are returned?

Reply:
No. This is probably me missing out on a rename. I'll change this one back.